### DISCUSSION TITLE
Homestead for tests

### DISCUSSION THREAD

**Discussion by CoolGoose** [spatie/laravel-query-builder#552] [Ideas]
Would a PR for having homestead for local development be useful ?
As an alternative I could just write the documentation on how to run the tests :P

**Comment by AlexVanderbist** [comment]
I'd accept a PR that adds some docs on sending PRs that are properly tested :p

**Reply by CoolGoose** [reply]
I can add that as well. I was thinking on using homestead since it would be easy then to just vagrant up, run the tests and add tests :P
